File: 01/1.js
const input = require("fs").readFileSync("input.txt").toString().split("\n").map(Number).filter(n => !isNaN(n))

for (const n of input) {
    if (input.includes(2020 - n)) {
        console.log(n * (2020 - n))
        break
    }
}

File: 06/1.hs
{-# LANGUAGE ViewPatterns #-}

import Data.List (sort,group)
import Data.List.Split (splitOn)

main = do
    input <- readFile "input.txt"
    let result = run input
    putStrLn $ show result

run = sum . map (length . group . sort . filter (/='\n')) . splitOn "\n\n"

